<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorHttpGetWidget</class>
 <widget class="QWidget" name="InspectorHttpGetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>478</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUrl">
     <item>
      <spacer name="horizontalSpacerUrl">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>43</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUrl">
       <property name="text">
        <string>URL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUrl"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutData">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutData">
         <item>
          <spacer name="verticalSpacerDataTop">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddHttpData">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>33</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Add new device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRemoveHttpData">
           <property name="minimumSize">
            <size>
             <width>33</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>33</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Remove device</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacerDataBottom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="HttpDataTreeBaseWidget" name="treeWidgetHttpData">
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>87</number>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerHttpGet">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HttpDataTreeBaseWidget</class>
   <extends>QTreeWidget</extends>
   <header>Inspector/HttpDataTreeBaseWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonAddHttpData</sender>
   <signal>clicked()</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>addRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRemoveHttpData</sender>
   <signal>clicked()</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>removeRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetHttpData</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidgetHttpData</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditUrl</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorHttpGetWidget</receiver>
   <slot>urlChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>flashlayerChanged(int)</slot>
  <slot>addRow()</slot>
  <slot>removeRow()</slot>
  <slot>urlChanged(QString)</slot>
  <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>triggerOnNextChanged(int)</slot>
 </slots>
</ui>
